client/tests/test-1.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
```

Comment:
What is test 1? What are you testing? Use meaningful name, help me understand the code.